applications/ocelloids_xcm_monitoring_service.md

```diff
@@ -0,0 +1,266 @@
+# Ocelloids XCM Transfer Monitoring Service
+
+- **Team Name:** SO/DA zone
+- **Payment Address:** Fiat 04.05.2023, 16:37
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 2
+
+## Project Overview
+
+This grant proposal is a follow-up to the Ocelloids Monitoring SDK, previously delivered and available here: https://github.com/w3f/Grant-Milestone-Delivery/pull/934
+
+The objective of this grant is to develop an open-source monitoring service using the Ocelloids SDK. This service will monitor XCM transfers across selected parachains. The primary purpose is to offer service providers integrating with a single chain (Asset Hub as starting point) and monitoring effects on other chains that are connected via HRMP and that use XCM as their message format. The service will support connecting to the configured networks through light clients in order to reduce infrastructure overhead for service providers. Users will have access to a self-hosted HTTP API to subscribe to XCM transfers and manage their subscriptions. A public Docker image will be published to facilitate service deployment.
+
+## Project Details
+
+The service will support bidirectional XCM reserve-based transfers between selected parachains.
```

Comment:
It will need to support both teleports and reserve-based transfers since the native assets of parachains (in the initial case ASTR and ACA) are teleported. Or, if that's too much, then probably state that it's only for non-teleported (chain, asset) pairs.

Reply:
We added support for asset teleports.